src/fe-text/irssi.c

```diff
@@ -352,6 +353,7 @@ int main(int argc, char **argv)
 	   you have to call setlocale(LC_ALL, "") */
 	setlocale(LC_ALL, "");
 
+	loglev = g_log_set_always_fatal(G_LOG_FATAL_MASK | G_LOG_LEVEL_CRITICAL);
```

Comment:
Wouldn't this mean that glib "critical" errors such as the #125 assertion failures would result in irssi crashing?

Or for example stuff like "Source ID ### was not found when attempting to remove it" which i've seen in a lot of glib apps after glib 2.40 was released